cql3/statements/create_keyspace_statement.hh

```diff
@@ -54,6 +55,8 @@ namespace statements {
 /** A <code>CREATE KEYSPACE</code> statement parsed from a CQL query. */
 class create_keyspace_statement : public schema_altering_statement {
 private:
+    static logging::logger _logger;
```

Comment:
I think it's fine to declare the logger as static in `.cc` file only, so it doesn't complicate the header. We do it everywhere else anyway.

Reply:
I was basing this decision on batch_statement.hh/.cc, which also adds its logger to the class. The 'statement' namespace is less polluted thanks to this.